packages/playwright-core/src/remote/playwrightConnection.ts

```diff
@@ -245,6 +250,18 @@ export class PlaywrightConnection {
     debugLogger.log('server', `[${this._id}] finished cleanup`);
   }
 
+  private logServerMetadata(message: object, direction: 'SEND' | 'RECV') {
+    const serverLogMetadata: { [key: string]: any } = {
+      timestamp: Date.now(),
```

Comment:
The server:metadata logs focuses more around the metadata for each message that is exchanged between the client and server. So it would make more sense if the logs contain the timestamp at which each message was sent instead of the the uptime in seconds from when the playwright server was started. 

The disadvantage with monotonic time is it requires additional work to figure out the exact timestamp at which the message was sent. The [initialTime](https://github.com/microsoft/playwright/blob/b66839b039713382f5b877551f4c6deb35240534/packages/playwright-core/src/utils/time.ts#L27) which is set once the playwright server is started needs to be logged to calculate the exact time at which the messages were exchanged b/w client and server.

One drawback of using wall time instead of monotonic time in this log is the server:channel logs print the monotonic time for each message. So for an end-user it could be confusing to see two different types of timestamp being logged. One possible way is renaming the field to wallTime to explicitly mention what it represents. 

Let me know your thoughts around this @dgozman

Reply:
Sure, if wall time is more convenient, let it be. Renaming to `wallTime` sounds good.